Beta_Integrations/integration-Vertica.yml

```diff
@@ -0,0 +1,178 @@
+description: Analytic database management software
+configuration:
+- display: Host (Vertica Server URL)
+  name: url
+  defaultvalue: myhost.example.com
+  type: 0
+  required: true
+- display: Database
+  name: database
+  defaultvalue: mydb
+  type: 0
+  required: true
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Port
+  name: port
+  defaultvalue: "5433"
+  type: 0
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+
+    import vertica_python
+    import datetime
+
+    ''' GLOBALS/PARAMS '''
+
+    USERNAME = demisto.params().get('credentials').get('identifier')
+    PASSWORD = demisto.params().get('credentials').get('password')
+    DATABASE = demisto.params().get('database')
+    PORT = int(demisto.params().get('port', 5433))
+    SERVER = demisto.params()['url'][:-1] if (demisto.params()['url'] and demisto.params()['url'].endswith('/')) else demisto.params()['url']
+    DB_PARAMS = {
+        'host': SERVER,
+        'port': PORT,
+        'user': USERNAME,
+        'password': PASSWORD,
+        'database': DATABASE,
+        'connection_timeout': 5
+    }
+
+    ''' HELPER FUNCTIONS '''
+
+    def convert_datetime_to_string(v):
```

Comment:
don't we have something like that im common server?

Reply:
we have ts to ds & date to ts, I needed dt to ds.